js/Popup.js

```diff
@@ -0,0 +1,120 @@
+const portfolio = document.getElementById('portfolio');
+
+const projects = [
+  {
+    projectName: '5554 ffffff',
+    projectExps: ['Backend Dev', '2015'],
+    projectImage: { alt: 'jfffjfjfj', imgUrl: '../assets/firstPjct.png' },
+    projectDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    techs: ['html', 'css', 'javascript'],
+    projectLinks: ['https://kigozijosephsuubi.me', 'https://github.com/Zeph180/my-portfolio-microverse'],
+    projectDescExt: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+
+  },
+  {
+    projectName: 'djdjdj',
+    projectExps: ['Backend Dev', '2015'],
+    projectImage: { alt: 'jfffjfjfj', imgUrl: '../assets/firstPjct.png' },
+    projectDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    techs: ['html', 'css', 'javascript'],
+    projectLinks: ['https://kigozijosephsuubi.me', 'https://github.com/Zeph180/my-portfolio-microverse'],
+    projectDescExt: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+
+  },
+];
+
+function breakTitle(title, element) {
+  const index = title.indexOf(' ');
+  const strOne = title.substr(0, index);
+  const strTwo = title.substr(index + 1);
+  const formatTitle = `${strOne}<br /> ${strTwo}`;
```

Comment:
[REQUIRED] Kindly fix the images for your projects section and popup section.
![image](https://user-images.githubusercontent.com/62938735/210761779-e4be3ffb-de0b-41b0-8079-b71705efd912.png)

![image](https://user-images.githubusercontent.com/62938735/210761847-f330bfce-be21-4c52-b37f-a3f6c73ee294.png)

Reply:
@fobadara 
Please remind me of what I'm supposed to fix because the comment seems to have no issue description. Thank you

Reply:
@fobadara 
I appreciate any suggestion on what exactly I'm supposed to do